internal/buildscripts/packaging/choco/splunk-otel-collector/splunk-otel-collector.nuspec

```
@@ -0,0 +1,50 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>splunk-otel-collector</id>
+    <version>0.0.0</version>
+    <description>
+The splunk-otel-collector is a metric-based agent written in Go that is used to monitor infrastructure and application services from a variety of environments.

```

I think we'd want a more generalized introduction to the utility:
```suggestion
Splunk OpenTelemetry Connector is a distribution of the OpenTelemetry Collector. It provides a unified way to receive, process, and export metric, trace, and log data for Splunk Observability Cloud.
```